.github/workflows/pushcodecommit.yml

```diff
@@ -0,0 +1,36 @@
+name: PushToCodeCommit
+
+on: workflow_dispatch
+
+jobs:
+  PushToCodeCommit:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with: 
+        role-to-assume: ${{secrets.AMI_MIRROR_ROLE}}
+        aws-region: us-west-2
+    - name: Delete shinkansen branch on codecommit repository
```

Comment:
Q: So we're deleting the shinkansen branch from the internal repo but then creating a new one (please correct me if I'm wrong)? Just curious, could we avoid deleting the branch and just push?

Reply:
Also, will there ever be case where we need to "rollback"/retrieve a past version of the branch? I'm worried if there's ever a case where we need to refer back to an older version.

Reply:
The reason I found it ok to delete the branch and recreate it is due to the fact that the github repository is the source of truth, whereas the codecommit repository is a mirror. It is effectively the same as mirroring only one branch, but due to internal reasons, we don't want any force pushes happening to the codecommit repository. 

So this leads into the second question. Since we push the whole branch from github (essentially mirror the branch) and we don't every do actual changes to the codecommit not in github (other than add one static file), no history will every be deleted from codecommit.